sleet.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'colorize', '~> 0.8.1'
   spec.add_dependency 'faraday', '>= 0.13.1', '< 1.1.0'
+  spec.add_dependency 'faraday_middleware'
```

Comment:
This new dependency is mostly tied to the version of `faraday` so seems unnecessary to restrict it.